docs/source/usersguide/input.rst

```diff
@@ -558,19 +559,156 @@ The following code snippet illustrates the instantiation of the ``CPUSolver`` fo
+
+.. code-block:: python
+
+  # Set the fixed source inside the source cell in group 1 to unity
+  solver.setFixedSourceByCell(source_cell, 1, 1.0)
+
+
+The equivalent code for setting the point source of all flat source regions within the source cell using ``setFixedSourceByFSR(...)`` is given below.
+
+.. code-block:: python
+
+  # Get the unique ID of the cell containing the point source
+  point_source_cell_id = source_cell.getId()
+
+  # Loop over all FSRs and test if the FSR is within the point source cell
+  for fsr_id in xrange(solver.getGeometry().getNumFSRs()):
+    cell = solver.getGeometry().findCellContainingFSR(fsr_id)
+
+    # If ithe FSR is within the point source cell, set the fixed source
+    if cell.getId() == point_source_cell_id:
+      solver.setFixedSourceByFSR(fsr_id, 1, 1.0)
+
+In this case, it is far simpler to set the fixed source by ``Cell``. However, there may be cases where the user may wish to set the fixed source within a ``Cell`` to different values. For instance, if the user wishes to model a continuously varying fixed source and there are multiple flat source regions within some ``Cell``, then for each flat source region within the cell the fixed source would need to be set individually.
+
+For instance, if the user desires to input a source based on the location within the geometry, setting the source with ``setFixedSourceByFSR(...)`` could be useful. In particular if a user has defined a fucntion ``source_distribution`` which provides the source over the flat source regions that depends on the coordinates within the geometry and the energy group, the following code will set the source to the appropriate values.
+
+.. code-block:: python
+
+  # Set the source every cell to the desired distribution
+  for fsr_id in xrange(solver.getGeometry().getNumFSRs()):
+
+    # Get the coordinates of some point within the FSR
+    pt = solver.getGeometry().getFSRPoint(fsr_id)
+    x_pt = pt.getX()
+    y_pt = pt.getY()
+
+    # Set the FSR source for every group
+    L = num_x * width_x / 2
+    H = num_y * width_y / 2
+    for g in range(materials['Water'].getNumEnergyGroups()):
+      group = g + 1
+      source_value = source_distribution(x_pt, y_pt, group)
+      solver.setFixedSourceByFSR(fsr_id, group, source_value)
+
+So far only the ``computeEigenvalue(...)`` routine has been introduced for solving neutron transport problems. The OpenMOC ``Solver`` has other solution options in addition to the eigenvalue solver which can be very useful for fixed source calculations. Specifically, the ``computeFlux(...)`` and ``computeSource(...)`` routines solve neutron transport over the problem without computing an eigenvalue.
+
+.. note:: The fixed source can **only** be set **after** ``TrackGenerator`` has generated tracks
```

Comment:
This warning is no longer valid per my latest commits today to #251. In order to make that PR work with the recently merged #242, I implement *deferred* fixed source assignments at simulation runtime. So the user simply has to generate tracks before running their calculation through the `Solver`, but *not* necessarily before assigning fixed sources.